src/Microsoft.CodeAnalysis.Razor/ViewComponentTagHelperDescriptorConventions.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.CodeAnalysis.Razor
     /// <summary>
     /// A library of methods used to generate <see cref="TagHelperDescriptor"/>s for view components.
     /// </summary>
-    internal static class ViewComponentTagHelperDescriptorConventions
+    public static class ViewComponentTagHelperDescriptorConventions
```

Comment:
Spoke offline. Filed an issue to track this. https://github.com/aspnet/Razor/issues/1383